### PR TITLE
negar/add_bem

### PR DESCRIPTION
**Pull request by negarn**
more BEM

### CODE REVIEW

File: src/sass/app_2/_common/layout/footer.scss [hunk 1]
```diff
@@ -122,7 +123,7 @@ footer {
             top: 0;
             left: 0;
         }
-        .inline-icon {
+        .ic-settings__icon {
```
Comment (kellybinary) [inline src/sass/app_2/_common/layout/footer.scss]: `&__icon` ?

Comment (negarn) [reply src/sass/app_2/_common/layout/footer.scss]: then I would have to change the class to `ic-settings--active__icon`, which is not the pattern we want to follow

Comment (negarn) [reply src/sass/app_2/_common/layout/footer.scss]: the icon for ic-settings--active has a different styling than non-active

Comment (kellybinary) [reply src/sass/app_2/_common/layout/footer.scss]: ahh, didn't see the `--active` please ignore this

**Pull request merged**